index.html

```diff
@@ -66,15 +82,30 @@ <h2 class="explore-the-data__header">Adjust the Slider to Explore the Data</h2>
       <button type="button" class="toggle__jobs" data-toggle="button">
         Show Jobs Heatmap
       </button>
+      <div id="phase-flex">
+        <div class="phase-col">
+          <button type="button" class="toggle__phase bp1 toggled__p1" data-toggle="button">
+            Phase 1 & 2
+          </button>
+        </div>
+        <div class="phase-col">
+          <button type="button" class="toggle__phase bp3 toggled__p3" data-toggle="button">
+            Phase 3
+          </button>
+          <button type="button" class="toggle__phase bp4 toggled__p4" data-toggle="button">
+            Phase 4
+          </button>
+        </div>
+      </div>
     </div>
   </div>
 </section>
 <section class="action-boxes">
   <div class="action-box">
-    <h2 class="action-box__title">Read the Full Report</h2>
-    <p class="action-box__text">Find the full report and technical appendix here.</p>
+    <h2 class="action-box__title">Read the Full Report (Phases 1 and 2)</h2>
+    <p class="action-box__text">Find the full phase 1 & 2 report and technical appendix here.</p>
```

Comment:
Not a blocker, but we could capitalize "phases" -> "Phases" to be consistent with other usage.